Add the home page "call to action" / signup pattern

carolinan opened the pull request:
Adding this pattern here for now, and we can discuss if it should be in the pattern directory or the theme.

I am concerned the text might indicate to some users that the theme has some sort of newsletter functionality.
Can we come up with something more neutral?

beafialho commented:
> I am concerned the text might indicate to some users that the theme has some sort of newsletter functionality.
> Can we come up with something more neutral?

Good point, how about this?

<img width="1032" alt="Captura de ecrã 2022-08-17, às 10 44 02" src="https://user-images.githubusercontent.com/65220155/185088261-431ae6f8-8335-4800-9e76-0622b5e9b422.png">

I noticed the line height seems a little too large to me, compared to the Figma specs:

<img width="1060" alt="Captura de ecrã 2022-08-17, às 10 47 52" src="https://user-images.githubusercontent.com/65220155/185089051-b1d1ae49-b8e1-445d-a6d1-3f12d42fdfa1.png">

carolinan commented:
When I updated the text, the question mark was on a third line. So I removed the 40% width of the left column.
That meant that the line did not extend as close to the text as in the design:

<img width="1059" alt="Screen Shot 2022-08-17 at 12 10 52" src="https://user-images.githubusercontent.com/7422055/185093953-796a2e87-fe79-478f-8deb-865e484c2e0b.png">

We should probably test with a row block too instead of a columns block.

carolinan commented:
I tested a row with a stack with the paragraph and button on the left, and the separator on the right, and it did not work, the separator is invisible.

<img width="1512" alt="row block with nested blocks" src="https://user-images.githubusercontent.com/7422055/185095633-0eeba226-9ec7-4a53-a3b3-580e3bd0f3a9.png">

beafialho commented:
> When I updated the text, the question mark was on a third line. So I removed the 40% width of the left column.
> That meant that the line did not extend as close to the text as in the design:

Thank you, I think that's ok.

mikachan commented on patterns/call-to-action.php:
```diff
@@ -0,0 +1,34 @@
+	<!-- wp:column -->
+	<div class="wp-block-column">
+		<!-- wp:paragraph {"style":{"typography":{"lineHeight":"1.2"}},"fontSize":"x-large"} -->
+		<p class="has-x-large-font-size" style="line-height:1.2">Got any book recommendations?</p>
```
Does this content need to be escaped/translated?

carolinan replied:
Yes!

mikachan reviewed (approved):
🚢🚢
